src/strutil.c

```diff
@@ -1031,3 +1031,79 @@ char *str_nexttok(char *buf, const char *delim, char **next) {
 
   return (char *) q;
 }
+
+/*
+ * This function implements the Damerau-Levenshtein algorithm to
```

Comment:
where is this from? Acknowledge authors and check whether licence is compatible with AVRDUDE's

Reply:
Forgot to add a URL to the source. It's from the git project repository. They use GPL 2 as well.
https://github.com/git/git/blob/master/levenshtein.c